docs/user/metrics/string.md

```diff
@@ -1,20 +1,62 @@
-# String
+# Strings
```

Comment:
Why plural and not singular?

IMO singular works for most metric types best (well, except for ... event(s) I guess?). We should at least be consistent.